Procfile

```diff
@@ -0,0 +1 @@
+web: npm run copyDemoPreferences && npm start --workspace=apps/server
```

Comment:
The widget urls probably won't work like this. The UI needs to be built for the server to work off it's assets

Reply:
This command is what's run after heroku finishes the build phase. The UI gets built during the build phase which comes from the package.json "build" script.